src/utilities/validation.ts

```diff
@@ -255,4 +255,28 @@ export const isHandle: ValidationMethodFactory = property => value => {
   }
 
   return `${property} must only contain lowercase letters, numbers, and dashes`
-}
\ No newline at end of file
+}
+
+export const isLowerCase: ValidationMethodFactory = property => value => {
+  if (isNullish(value) || isEmptyString(value)) {
+    return true
+  }
+
+  if (typeof value === 'string' && value === value.toLowerCase()) {
+    return true
+  }
+
+  return `${property} must be lowercase`
+}
+
+export const isUpperCase: ValidationMethodFactory = property => value => {
+  if (isNullish(value) || isEmptyString(value)) {
+    return true
+  }
+
+  if (typeof value === 'string' && value === value.toUpperCase()) {
+    return true
+  }
+
+  return `${property} must be uppercase`
+}
```

Comment:
Technically these are out of scope since I ended up using the `isHandle` validator instead but they felt like nice to haves (`isHandle` is doing lowercase checks under the hood). I don't have an immediate use for them though so happy to remove from this PR if anyone feels strongly.